src/components/panels/database/DatabasePanel.tsx

```diff
@@ -97,17 +95,12 @@ function DatabasePanel({ height, fen }: { height: number; fen: string }) {
 
   const tab = useAtomValue(currentTabAtom);
 
-  const dbOptionsKey = match(db)
-    .with("local", () => "0")
-    .with("lch_all", () => `1_${JSON.stringify(lichessOptions)}`)
-    .with("lch_master", () => `2_${JSON.stringify(masterOptions)}`)
-    .exhaustive();
   const {
     data: openingData,
     isLoading,
     error,
-  } = useSWR([dbType, query, dbOptionsKey], async ([dbType, query]) => {
-    return fetchOpening(query, dbType, tab?.value || "", lichessOptions, masterOptions);
```

Comment:
The reason for this `dbOptionsKey` was so the cache is only relevant to the selected DB e.g. changing options for `lch_all` (without changing the position/fen) won't invalidate the cache for `lch_master`
- Set up a position
- View `lch_master` DB then `lch_all` DB
- Change one of the `lch_all` options
- Switch back to `lch_master` - should still have the same cached data rather then re-requesting

Reply:
Oh I see, that makes sense, but it would still fetch again if the local options changed from exact to partial for example. I'll be refactoring this a bit so that local options and lichess options are more consistent.